ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/analytics/model/TutorialAnalyticsActionEvent.kt

```diff
@@ -0,0 +1,16 @@
+package org.cru.godtools.tutorial.analytics.model
+
+import org.cru.godtools.analytics.model.AnalyticsActionEvent
+import org.cru.godtools.analytics.model.AnalyticsSystem
+
+internal const val ACTION_TUTORIAL_ONBOARDING_TRAINING = "onboarding_more"
+internal const val ACTION_TUTORIAL_FINISH = "onboarding_start"
+
+class TutorialAnalyticsActionEvent(action: String) : AnalyticsActionEvent(null, action) {
+    override fun isForSystem(system: AnalyticsSystem): Boolean {
+        return system == AnalyticsSystem.ADOBE || system == AnalyticsSystem.FACEBOOK
+    }
+
+    override val adobeSiteSection get() = ADOBE_SITE_SECTION_TUTORIAL
+    override fun getAttributes() = mapOf(action to 1)
```

Comment:
I'm mostly curious about what the action attributes should look like, this is the first instance where we use the same string for the action and the attribute. So either they have changed how they are doing it or you are doing something unnecessry or incorrect